docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -1143,50 +1143,7 @@ The *fhir-bulkimportexport-webapp* module is a wrapper for the whole BulkData we
     </webApplication>
 ```
 
-BulkData web application writes the exported FHIR resources to an IBM Cloud Object Storage (COS) or Amazon S3 bucket, as configured in the per-tenant bulkdata.json configuration file. The bulkdata.json file is stored in the tenant configuration directory of each fhir-server instance. The following is a bulkdata.json which is configured to export the FHIR resources into fhir-bulkdata-sample bucket of IBM COS:
-
-```json
-{
-  "applicationName": "fhir-bulkimportexport-webapp",
-  "moduleName": "fhir-bulkimportexport.war",
-  "jobParameters": {
-    "cos.bucket.name": "fhir-bulkdata-sample",
-    "cos.location": "us",
-    "cos.endpointurl": "https://s3.us-south.cloud-object-storage.appdomain.cloud",
-    "cos.credential.ibm": "Y",
-    "cos.api.key": "xxxxxxxxxxxxxxxxxxxxxxx",
-    "cos.srvinst.id": "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx",
-  },
-  "implementation_type": "cos",
-  "batch-uri": "https://localhost:9443/ibm/api/batch/jobinstances",
-  "batch-user": "fhiradmin",
-  "batch-user-password": "xxxxxxxxxxxx",
-  "batch-truststore" : "resources/security/fhirTruststore.jks",
-  "batch-truststore-password" : "xxxxxxxxxxxx",
-  "serverHostname" : "localhost:9443",
-  "contextRoot" : "/fhir-server/api/v4"
-}
-```
```

Comment:
I agree this content is outdated; now the config data exists in fhir-server-config.json instead of a separate bulkdata.json; however, I think maybe we should leave a sample config here just to illustrate the properties (and refer to section 5 for the full list of properties).

Reply:
good point, let me add one

Reply:
done